azurerm/resource_arm_virtual_network.go

```diff
@@ -300,6 +288,51 @@ func getVirtualNetworkProperties(ctx context.Context, d *schema.ResourceData, me
 	// finally; return the struct:
 	return properties, nil
 }
+func flattenVirtualNetworkSubnets(input *[]network.Subnet) *schema.Set {
+	results := &schema.Set{
+		F: resourceAzureSubnetHash,
+	}
+
+	if subnets := input; subnets != nil {
+		for _, subnet := range *input {
+			output := map[string]interface{}{}
+
+			if name := subnet.Name; name != nil {
+				output["name"] = *name
+			}
+
+			if props := subnet.SubnetPropertiesFormat; props != nil {
+				if prefix := props.AddressPrefix; prefix != nil {
+					output["address_prefix"] = *props.AddressPrefix
```

Comment:
I believe you could go `output["address_prefix"] = *prefix` here

Reply:
👍 fixed